bundle-workflow/Jenkinsfile

```diff
@@ -24,6 +24,11 @@ pipeline {
             }
         }
         stage('build') {
+            agent {
+                node {
+                    label 'Jenkins-Agent-al2-x64-m5xlarge'
+                }
+            }
```

Comment:
Why do we need to add another node here?

Reply:
Since the top agent is `none` it has to be specified for any usages, see [agent directive](https://www.jenkins.io/doc/book/pipeline/syntax/#agent). The curl command needs to execute somewhere - and it would be better that it wasn't the main host.

Note: a very tiny docker image would be great to switch to when available

Reply:
Love that we're using a m5xlarge to send a notification :)